### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by t3ddy33**

### CODE REVIEW

**Comment by LiamBurgess18015001** [comment]
:eye:

**Review by LiamBurgess18015001 — APPROVED** [review]
the return false for no token can be avoided by setting anything for the token before the test but otherwise it looks good :+1:

File: master/libs/thingsboard/thingsboard-user/src/lib/thingsboard-thingsboard-user.service.spec.ts [hunk 1]
```diff
@@ -14,30 +18,90 @@ describe('ThingsboardThingsboardUserService', () => {
...
+        "refreshToken": "eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJ0ZW5hbnRAdGhpbmdzYm9hcmQub3JnIi..."
+      },
+      headers: {},
+      config: {},
+      status: 200,
+      statusText: 'OK'
+    }
+    jest.spyOn(httpService, 'post').mockImplementationOnce(() => of(result));
+    const login = await service.login('reserveAdmin@reserve.com', 'reserve');
+    expect(login['data']['token']).toBeDefined;
```
Comment (Isakvdw) [inline master/libs/thingsboard/thingsboard-user/src/lib/thingsboard-thingsboard-user.service.spec.ts:41]: * toBeDefined()

Comment (LiamBurgess18015001) [reply master/libs/thingsboard/thingsboard-user/src/lib/thingsboard-thingsboard-user.service.spec.ts:41]: will fix in the next PR

**Pull request merged**